tests/test_e2e.py

```diff
@@ -139,7 +139,11 @@ def test_vllm_e2e(self, protocol, setup_and_teardown):
         pid = utils.run_server()
         setup_and_teardown.pid = pid
         # vLLM will download the model on the fly, so give it a big timeout
-        utils.wait_for_server_ready(timeout=300)
+        # TODO: Consider one of the following
+        # (a) Pre-download and mount larger models in test environment
+        # (b) Download model from HF for vLLM at import step to remove burden
+        #     from server startup step.
```

Comment:
If we go with (a) will this leave behind artifacts in the runner or will it be cleared automatically?

Reply:
For pre-downloaded mount, I was thinking of having it in the /data NFS mount or something, not directly on runner.